### PR TITLE
Add JSON dump functionality

### PR DESCRIPTION
**Pull request by ajread4**
Added a script called evtx_dump_json.py that outputs evtx files in JSON format or to a file in a JSON array. Primary use case is for easy ingestion into Pandas DataFrames for graph learning, data science, etc. Updated README with reference to new file.

### CODE REVIEW

File: scripts/evtx_dump_json.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+#
+#   Purpose: User can dump evtx data into JSON format to either the command line or a JSON file in new line delimited format/JSON array.
+#   Details: The JSON object is created with only the EventRecordID from the System section of the evtx XML and all of the information within the EventData section.
+
+import Evtx.Evtx as evtx
+import Evtx.Views as e_views
+
+# Added packages
+import os
+import xmltodict
```
Comment (williballenthin) [inline scripts/evtx_dump_json.py:13]: please update `setup.py` with the new dependency on `xmltodict`

**Review by williballenthin — APPROVED** [review]
one suggestion, then lets merge!

File: scripts/evtx_dump_json.py (hunk 1 above)
Comment (ajread4) [reply scripts/evtx_dump_json.py:13]: Added to the install_requires section! 

` install_requires=[
            'six',
            'hexdump==3.3',
            'xmltodict==0.13.0', #added deps for evtx_dump_json.py script
               
            # pin deps for python 2, see #67
            'more_itertools==5.0.0',
            'zipp==1.0.0',
            'configparser==4.0.2',
            'pyparsing==2.4.7',
            ],`

**Comment by williballenthin** [comment]
thank you!

**Pull request merged**